### PR TITLE
User story 5

### PR DESCRIPTION
**Pull request by jhgould**
Add user story 5-7

### CODE REVIEW

File: spec/features/reviews/update_review_spec.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      click_link "update_review_#{@review_1.id}"
+      expect(current_path).to eq("/reviews/#{@review_1.id}/edit")
+
+    end
+
+    it "user can update from update form" do
+      visit "/reviews/#{@review_1.id}/edit"
+      # expect(page).to have_content("#{@review_1.title}")
+      # expect(page).to have_content("#{@review_1.rating}")
+      # expect(page).to have_content("#{@review_1.content}")
```
Comment (jhgould) [inline spec/features/reviews/update_review_spec.rb:24]: We might need to circle back to this at some point and test that the fields are auto-populating.

File: spec/features/reviews/update_review_spec.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      expect(current_path).to eq("/shelters/#{@shelter.id}")
+      expect(page).to have_content("Average Shelter")
+      expect(page).to have_content("Rating:3")
+      expect(page).to have_content("Only had cats")
+
+    end
+    it "user will get flash notice if required fields are not filled in" do
+      visit "/reviews/#{@review_1.id}/edit"
+
+      fill_in "title", with: ""
```
Comment (jhgould) [inline spec/features/reviews/update_review_spec.rb:41]: Had to fill in `title` with an empty string to make sure that something was not auto-populated.

File: app/views/reviews/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<h1>Edit Review</h1>
+
+<%= form_tag "/reviews/#{@review.id}", method: :patch do  %>
+  <%= label_tag :title, "Title: " %>
+  <%= text_field_tag :title, "#{@review.title}" %>
```
Comment (jhgould) [inline app/views/reviews/edit.html.erb:5]: The auto-populating text is not being picked up by the test. It works in the app.

File: app/views/reviews/edit.html.erb [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  <%= label_tag :rating, "Rating:" %>
+  <%= number_field_tag :rating, min: 1, max: 5 %>
```
Comment (jpc20) [inline app/views/reviews/edit.html.erb:7]: Let's see if we can get the min and max to work on the app.

File: spec/factories.rb [hunk 5]
```diff
@@ -2,4 +2,12 @@
   factory :shelter do
     name { "Angels With Paws" }
   end
+  factory :review do
+    title { "Great Shelter" }
+    rating {5}
+    content {"This place was awesome!"}
+    association :followed_by_shelter, factory: :shelter
```
Comment (jpc20) [inline spec/factories.rb:9]: We couldn't figure out how to get the review factory working because we have to include shelter as a foreign key.

**Pull request merged**